### ISSUE TITLE
レビュー：衍字いくつか

### ISSUE THREAD

**Issue by jidaikobo-shibata** [waic/wcag21#1631]
https://dev.waic.jp/translations/WCAG21/Techniques/failures/F34.html
> 又**はは**固定フォントから可変フォントに変更される、

WGAC 2.0で同様:
https://waic.jp/translations/WCAG-TECHS/failures.html
https://waic.jp/translations/WCAG-TECHS/complete.html
https://waic.jp/translations/WCAG-TECHS/F34.html

----

https://dev.waic.jp/translations/WCAG21/Techniques/general/G152.html
> アニメーションの再生時間**をを**表示する。

WGAC 2.0で同様:
https://waic.jp/translations/WCAG-TECHS/general.html
https://waic.jp/translations/WCAG-TECHS/complete.html
https://waic.jp/translations/WCAG-TECHS/G152.html

----

https://waic.jp/translations/WCAG21/Techniques/css/C20.html （**devでは、修正済み！**）
> より効率よくコンテンツ**をを**閲覧

https://dev.waic.jp/translations/WCAG21/Techniques/css/C20.html
では修正されているが、WCAG 2.0では放置:
https://waic.jp/translations/WCAG-TECHS/css.html
https://waic.jp/translations/WCAG-TECHS/C20.html

----

20230306 追加
https://dev.waic.jp/translations/WCAG21/Understanding/pointer-gestures.html
> スライダーが作動しないことが**ああ**る (実装による)

----

今回のレビューではないけど、WCAG 2.0の衍字。

https://waic.jp/translations/WCAG-TECHS/html.html
https://waic.jp/translations/WCAG-TECHS/H69.html
> 2 の結果が真であること**をを**確認する。

https://waic.jp/translations/WCAG20/Overview_20101125.html
古いドキュメントなので、別の検討が必要か。
> 又**はは**14ポイントの太字

https://waic.jp/translations/UNDERSTANDING-WCAG20/minimize-error-cues.html
> より大**きき**なコントロールのクリック範囲

**Comment by jidaikobo-shibata** [comment]
> スライダーが作動しないことがあある

を追加しました。
